### DISCUSSION TITLE
JedisSentinelPool supports ssl parameters?

### DISCUSSION THREAD

**Discussion by saiprasad91** [xetorthio/jedis#2569] [Q&A]
Hi All

we are using jedis 3.6.0 lib in our application to connect with REDIS 6.2. As part of rthis we would like to try out ACL & TLS options.
we find JedisSentinelPool doesn't support ssl arguments from below url!!!
https://javadoc.io/doc/redis.clients/jedis/3.6.0/redis/clients/jedis/JedisSentinelPool.html.

Looking forward for the solution to connect sentinels over TLS....

Also need pointers on alternatives to JedisSentinelPool deprecation.

Thanks
Sai Prasad

**Comment by sazzad16** [comment]
- `JedisSentinelPool(String masterName, Set<HostAndPort> sentinels, org.apache.commons.pool2.impl.GenericObjectPoolConfig<Jedis> poolConfig, JedisClientConfig masteClientConfig, JedisClientConfig sentinelClientConfig)`
- `JedisSentinelPool(String masterName, Set<HostAndPort> sentinels, org.apache.commons.pool2.impl.GenericObjectPoolConfig<Jedis> poolConfig, JedisFactory factory, JedisClientConfig sentinelClientConfig)`

JedisSentinelPool can be configured with SSL through above two constructors. Check [the test codes](https://github.com/redis/jedis/blob/6e9ba1bd83402ce46db976c56d84906779322995/src/test/java/redis/clients/jedis/tests/SSLJedisSentinelPoolTest.java) for more help.
